src/nvim/sign.c

```diff
@@ -1374,16 +1365,15 @@ void sign_getlist(const char_u *name, list_T *retlist)
 list_T *get_buffer_signs(buf_T *buf)
   FUNC_ATTR_NONNULL_RET FUNC_ATTR_NONNULL_ALL FUNC_ATTR_WARN_UNUSED_RESULT
 {
-  signlist_T  *sign;
-  dict_T  *d;
-  list_T *const l = tv_list_alloc(kListLenMayKnow);
+  signlist_T *sign;
+  dict_T *dict;
+  list_T *const list = tv_list_alloc(kListLenMayKnow);
```

Comment:
There are at least 2 reason not to rename things like this:

1. The scope is very small. Using long names for small scopes is more harmful than useful.
2. It is unidiomatic for the Vim/Nvim codebase. We are not going to rename every `d` to `dict`, and it doesn't add useful information anyway. `d` is almost always a dict. `l` is very often a list.

Reply:
Ok, I am not fully convinced (entering codebase with variable `d` vs `dict` makes difference for me), but I reverted the naming changes ;)

Reply:
the rename is still present

Reply:
@justinmk Now should be good